assessments/KRAssessment/seeds/objectiveAssessments.csv

```diff
@@ -0,0 +1,5 @@
+identificationCode,assessmentIdentifier,namespace,description
+SFScore,KRA,uri://ed.sc.gov/KRA,Social Foundations Score
```

Comment:
typically the objective assessment would be like 'Social Foundations' and then there would be a score within that called 'Social Foundations Score'